ament_clang_format/cmake/ament_clang_format_lint_hook.cmake

```diff
@@ -0,0 +1,14 @@
+file(GLOB_RECURSE _source_files FOLLOW_SYMLINKS
+  "*.c"
+  "*.cc"
+  "*.cpp"
+  "*.cxx"
+  "*.h"
+  "*.hh"
+  "*.hpp"
+  "*.hxx"
+)
```

Comment:
These hook files are only used through `ament_lint_auto` which itself gets triggered by the `ament_package` extension point. So this will always be in the root of the package.